src/PointwiseFunctions/AnalyticSolutions/NewtonianEuler/IsentropicVortex.hpp

```diff
@@ -110,7 +126,9 @@ class IsentropicVortex {
 
   using options = tmpl::list<AdiabaticIndex, Center, MeanVelocity,
                              PerturbAmplitude, Strength>;
-  static constexpr OptionString help = {"Newtonian Isentropic Vortex."};
+  static constexpr OptionString help = {
+      "Newtonian Isentropic Vortex."
```

Comment:
If you want this displayed on two lines in the terminal you need a `\n`, otherwise it looks like it would fit on one line